pyproject.toml

```diff
@@ -23,6 +23,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
```

Comment:
Any reason 3.6 is specified? Its got a relatively short lifecycle compared to 3.7/8 https://devguide.python.org/#status-of-python-branches

Reply:
I agree with this. Prob worth bumping it up if possible.